doc/api/fs.md

```diff
@@ -238,6 +242,10 @@ on Unix systems, it never was.
 
 Emitted when the WriteStream's file is opened.
 
+### Event: 'close'
+
+Emitted when the WriteStream's underlying file descriptor has been closed. Comes from fs.close() method. 
```

Comment:
@nodejs/streams can we please check this? I _thought_ `'close'` was for `Readable` and `'finish'` was for `'Writable'`. However, in `'fs'` we have `'close'` for `'Writable'` as well. I do not think custom stream events are that helpful in understanding, so maybe we might want to rationalize this (or document it in streams).